app/resources/css-default.template

```diff
@@ -765,6 +790,16 @@ label.option {
   width: 100%;
 }
 
+#photo_upload {
```

Comment:
yes,
this is because that I don't want the under border for the upload part.
If remove this:
![underline](https://cloud.githubusercontent.com/assets/15961861/18829058/8e372a0c-8415-11e6-9a31-d6309097a1db.png)